app/controllers/spree/admin/images_controller_decorator.rb

```diff
@@ -0,0 +1,15 @@
+module Spree
+  module Admin
+    class ImagesControllerDecorator
+      after_action :update_mailchimp_product, only: [:create, :update]
```

Comment:
it will not work this way. Also, why do we want to make it in the controller, instead of a model?